sklearn/utils/__init__.py

```diff
@@ -19,6 +19,7 @@
 
 import warnings
 import numpy as np
+from distutils.version import LooseVersion
```

Comment:
I think distutils is deprecated, isn't it? I'm not sure if we can keep using it.

Reply:
[That's right](https://www.python.org/dev/peps/pep-0632/), yet it is being still use with [a modification of `packaging.version.parse`](https://github.com/scikit-learn/scikit-learn/blob/5efcb10c3e9f87103c404c32a036beb623182c83/sklearn/externals/_packaging/version.py#L65). I'll use this latter instead.

Reply:
I know it's used, but I think we should fix it, rather than introducing it in more places :D And I'm genuinely curious to see what we're supposed to do instead

Reply:
This can be done in a dedicated PR then. :)